### PR TITLE
document new version of the Prometheus plugin in 2.8

### PR DESCRIPTION
**Pull request by kikito**
It adds one extra field to the plugin.

The pull request which introduced the change is https://github.com/Kong/kong/pull/8387

Internal Tracking in FT-2426

### CODE REVIEW

**Comment by kikito** [comment]
Apologies for the late sending of this change; it arrived as an exception to the regular release process.

File: app/_hub/kong-inc/prometheus/index.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ Access-Control-Allow-Origin: *
 # TYPE kong_bandwidth counter
 kong_bandwidth{type="egress",service="google",route="google.route-1"} 1277
 kong_bandwidth{type="ingress",service="google",route="google.route-1"} 254
+# HELP kong_nginx_timers Number of nginx timers
+# TYPE kong_nginx_timers gauge
+kong_nginx_timers{state="running"} 3
+kong_nginx_timers{state="pending"} 1
```
Comment (kikito) [inline app/_hub/kong-inc/prometheus/index.md:104]: @limkevinkuan could you please confirm that these lines are correct?

**Comment by lena-larionova** [comment]
Was the plugin version bumped twice for 2.8? There was another change that came in earlier, which bumped the version to 1.5: https://github.com/Kong/docs.konghq.com/pull/3666

File: app/_hub/kong-inc/prometheus/index.md (hunk 1 above)
Comment (limkevinkuan) [reply app/_hub/kong-inc/prometheus/index.md:104]: Yep, they're good.

**Comment by kikito** [comment]
@lena-larionova You are right. This is (yet another) example of why our current way of versioning plugins is confusing.

Since in our source code we have 1.6.0 now, and I want the docs to be in sync with the source code, I have removed 1.5.0 and left 1.6.0 only.

**Comment by lena-larionova** [comment]
Hoping for Gateway-aligned plugin versioning with 3.0 or soon after, fingers crossed.

**Pull request merged**